tests/pytorch/mock_sparse/test_sddmm.py

```diff
@@ -0,0 +1,48 @@
+import numpy as np
+import pytest
+import dgl
+import dgl.backend as F
```

Comment:
You can directly use PyTorch here because the file is under tests/pytorch

Reply:
Done.